scipy/signal/signaltools.py

```diff
@@ -1786,8 +1786,8 @@ def resample(x, num, t=None, axis=0, window=None):
 
     See also
     --------
-    decimate
-    resample_poly
+    decimate : Downsample the signal after applying an anti-aliasing filter.
```

Comment:
As far as I can tell now `decimate` is just a special case of resampling, although it does allow for IIR antialiasing filters whereas `resample` and `resample_poly` do not. I'd just say `downsample a signal` or something and leave it at that